middleware/setBananaI18n.js

```diff
@@ -61,11 +61,11 @@ module.exports = (req, res, next) => {
 
   const translatedLocales = [];
   let activeLocale;
-
+  // Do we still need the for loop?
```

Comment:
we need to find the activeLocale still but don't need to translate it, we'd just need to also change the view that creates the locale dropdown

Reply:
Oh ok, so for loop stays to check for active locales but no more translation needed right? I'll change the code in a bit